src/main/java/uk/gov/hmcts/ccd/data/user/DefaultUserRepository.java

```diff
@@ -88,6 +88,12 @@ public IdamUser getUser() {
         return toIdamUser(userInfo);
     }
 
+    @Override
+    public IdamUser getUser(String userToken) {
+        UserInfo userInfo = securityUtils.getUserInfo(userToken);
+        return toIdamUser(userInfo);
+    }
+
     @Override
```

Comment:
May we you can add a test in DefaultUserRepositoryTest Optional It is not mandatory if sonar already passed. :)

Reply:
Added test case in DefaultUserRepositoryTest